src/main/java/org/hydev/mcpm/client/export/ImportResult.java

```diff
@@ -0,0 +1,61 @@
+package org.hydev.mcpm.client.export;
+
+import java.util.Map;
+
+
+/**
+ * Class storing results of the
+ */
+public class ImportResult {
+    private final State state;
+    private final Map<String, Boolean> installResults;
+
+    /**
+     * Result for importing plugins
+     *
+     * @param installResults Result of each install
+     */
+    public ImportResult(Map<String, Boolean> installResults) {
+        this.installResults = installResults;
+        boolean success = true;
```

Comment:
These are odd variable names. Consider moving them into the for loop and renaming them to refer to explicit conditions instead.